tasks/build/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = function (grunt) {
       'clean:build',
       'clean:target',
       '_build:downloadNodeBuilds:start',
+      '_build:downloadNodeBuilds:finish',
```

Comment:
i think that download usually takes less than 1 minute and the rest of the build process takes around 10 minutes. so it doesn't feel right to wait for 10 minutes just to find out that my node binaries are corrupt. however we could put it a bit further down the line. in my opinion joining the tasks into one would be the cleanest solution (as download is usually fast).

let me know